core/src/test/java/org/hibernate/ogm/test/util/impl/TupleContextHelperTest.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.test.util.impl;
+
+import static org.fest.assertions.Assertions.assertThat;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+import org.hibernate.HibernateException;
+import org.hibernate.engine.spi.SessionImplementor;
+import org.hibernate.ogm.dialect.spi.TupleContext;
+import org.hibernate.ogm.model.impl.DefaultEntityKeyMetadata;
+import org.hibernate.ogm.model.key.spi.EntityKeyMetadata;
+import org.hibernate.ogm.model.spi.EntityMetadataInformation;
+import org.hibernate.ogm.util.impl.TupleContextHelper;
+import org.hibernate.ogm.utils.OgmTestCase;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+/**
+ * Test the {@link TupleContextHelper} utility classe.
+ * <p>
+ * An {@link EntityMetadataInformation} will be null when there is more than one entity is involved.
```

Comment:
there is more than one entity is involved -> there is more than one entity type involved